Test/TilemapManagerTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using FluentAssertions;
+using Hel.Tiled;
+using NUnit.Framework;
+
+namespace Test {
+
+    [TestFixture]
+    public class TilemapManagerTests {
+
+        private TilemapManager _tilemapManager;
+        
+        [SetUp]
+        public void Setup() {
+            _tilemapManager = new TilemapManager(Loader.LoadTilemap("./MapFiles/TilemapManagerTests.json"));
+        }
+
+        [Test]
+        [TestCase("ObjectA", 1)]
+        // TODO: Test fails, fix bug. 
```

Comment:
do you know why is the test failing?

Reply:
I wrote the issue here https://github.com/HelEngine/Hel.Tiled/issues/7 related to the bug. 

The tests are failing because the `GetObjects` method isn't currently able to detect or find nested layers, it only iterates over top-level layers.